pages/invite.js

```diff
@@ -92,18 +110,23 @@ class Invite extends React.Component {
 
     return (
       <div style={{ padding: '30px 50px' }} className="has-text-centered">
-        <h1 className="title is-1">Accept Invitation</h1>
+        <h2 className="title is-2">&#x1F389; Invitation for {club.name} &#x1F389;</h2>
```

Comment:
Maybe add a comment that this is the confetti emoji

Reply:
Comment added!